spec/xmlspec-ttml1.xsl

```diff
@@ -612,6 +608,44 @@ width: 100%;
   </xsl:choose>
 </xsl:template>
 
+<!-- implreploc: implementation report for this spec -->
+<!-- called in a <dl> context from header -->
+<xsl:template match="implreploc">
+  <xsl:choose>
+    <xsl:when test="count(loc) &gt; 1">
+      <xsl:for-each select="loc">
+        <dt>
+          <xsl:apply-templates select="node()"/>
+        </dt>
+        <dd>
+          <a href="{@href}">
+            <xsl:value-of select="@href"/>
+          </a>
+        </dd>
+      </xsl:for-each>
+    </xsl:when>
+    <xsl:otherwise>
+      <dt>Implementation report:</dt>
+      <dd>
+        <xsl:apply-templates/>
+      </dd>
+    </xsl:otherwise>
+  </xsl:choose>
+</xsl:template>
+
+<!-- ghrepoloc: participate section -->
+<!-- called in a <dl> content from header -->
+<xsl:template match="ghrepoloc">
+  <xsl:apply-templates/>
+</xsl:template>
+<xsl:template match="ghloc">
+  <dt>Participate:</dt>
+  <dd><a href="https://github.com/{@repo}/">GitHub <xsl:value-of select="@repo"/></a></dd>
+  <dd><a href="https://github.com/{@repo}/issues/">File a bug</a></dd>
+  <dd><a href="https://github.com/{@repo}/commits/{@branch}">Commit history</a></dd>
+  <dd><a href="https://github.com/{@repo}/pulls/">Pull requests</a></dd>
```

Comment:
This is too much for my tastes: it takes 4 lines that all basically take the reader to the GitHub repo. Is there a reason we need the lines other than the GitHub repo link?